stacker_blueprints/ecs.py

```diff
@@ -375,10 +376,29 @@ def environment(self):
 
         return env_list
 
+    @property
+    def log_group(self):
+        return self.get_variables()["LogGroup"]
+
     @property
     def log_configuration(self):
-        log_config = self.get_variables()["LogConfiguration"]
-        return log_config or NoValue
+        """
+        If LogConfiguration is given, go with it. Otherwise if LogGroup is given,
```

Comment:
nit: This comment isn't quite right - we don't let LogConfiguration be given any longer.